RELEASE.md

```diff
@@ -41,6 +41,12 @@
 - ...
 -->
 
+## Versione X.X.X (dd/mm/yyyy)
+
+### Fix
+
+- Aggiunto titolo della sorgente per i singoli card - feed RSS
```

Comment:
Modificare in : Nei template dei feed RSS ora viene mostrata la sorgente se presente.